### PR TITLE
Nexus

### PR DESCRIPTION
**Pull request by sherl0cks**
#### What does this PR do?

- A quick and dirty role to configure Nexus repos
- Tests that support API object updates for nexus to be added to our core apps.

#### Which tests illustrate how this code works?
Look at the diff

#### Who would you like to review this?
/cc @oybed

### CODE REVIEW

**Comment by oybed** [comment]
Change looks good (I haven't tried it yet though), but I'm wondering if you could add a generic playbook at the top level playbook directory in the repo that accepts parameters (just like the `load_infra.yml` playbook) - basically a generalized version of your `cdk_create_nexus.yml`. I would like to see us build out of a set of playbooks that can be used for specific purposes like that.

File: roles/configure-nexus/tests/ocp-sandbox-test.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+# This test covers the full feature set provided by the role
+
+- name: "Configure Nexus"
+  hosts: nexus
+
+  vars:
+    nexus_url: http://nexus3-nexus.apps.s2.core.rht-labs.com
```
Comment (oybed) [inline roles/configure-nexus/tests/ocp-sandbox-test.yml]: could we remove the `rht-labs.com` portion here ? ... maybe just use `example.com` instead?

**Comment by sherl0cks** [comment]
@oybed updated per your request. I moved the variable structure to use inventory variables to support running this against multiple hosts

**Comment by oybed** [comment]
@sherl0cks thank you - looks good. merging.

**Pull request merged**